schemas/com.mbta.ctd.glides.operator_signed_in.v1.json

```diff
@@ -4,21 +4,32 @@
   "description": "Operator Signed In",
   "type": "object",
   "properties": {
-    "type": { "const": "com.ctd.glides.operator_signed_in.v1.json" },
+    "type": { "const": "com.mbta.ctd.glides.operator_signed_in.v1" },
     "specversion": { "const": "1.0" },
-    "source": { "const": "glides.mbta.com" },
-    "id": { "$ref": "glides-events-schema.json#/properties/id" },
-    "time": { "$ref": "glides-events-schema.json#/$defs/timestamp" },
+    "source": { "$ref": "glides-events.json#/properties/source" },
+    "id": { "$ref": "glides-events.json#/properties/id" },
+    "time": { "$ref": "glides-events.json#/$defs/timestamp" },
     "data": {
       "type": "object",
       "properties": {
-        "metadata": { "$ref": "glides-events-schema.json#/$defs/metadata" },
-        "operator": { "$ref": "glides-events-schema.json#/$defs/operator" },
-        "signedInAt": { "$ref": "glides-events-schema.json#/$defs/timestamp" },
-        "signature": { "$ref": "glides-events-schema.json#/$defs/signature" }
+        "metadata": { "$ref": "glides-events.json#/$defs/metadata" },
+        "operator": { "$ref": "glides-events.json#/$defs/operator" },
+        "signedInAt": { "$ref": "glides-events.json#/$defs/timestamp" },
+        "signature": { "$ref": "#/$defs/signature" }
       },
       "required": ["metadata"]
     }
   },
-  "required": ["type", "specversion", "source", "id", "time", "data"]
+  "required": ["type", "specversion", "source", "id", "time", "data"],
+  "$defs": {
+    "signature": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "examples": ["tap", "type"]
```

Comment:
question: did you mean for these to be `const` values instead? The documentation doesn't seem to indicate that this is a free-text field.

Reply:
It is supposed to be free text so that any changes to how we represent signatures don't cause a breaking change, and because supporting arbitrary strings here isn't a big burden on a client.